app/services/foreman/plugin.rb

```diff
@@ -396,5 +417,47 @@ def extend_template_helpers_by_module(mod)
       end
       allowed_template_helpers(*(mod.public_instance_methods - Module.public_instance_methods))
     end
+
+    def add_manager_roles(resources, opts)
+      manager = Role.find_by :name => "Manager"
+      return if !manager || resources.empty?
+      add_permissions manager, resources, opts
+    end
```

Comment:
Is the idea that for this role, all "assumed" permissions are added for a given resource? Those being:

 * view_
 * edit_
 * create_
* delete_